src/modal/modal.component.ts

```diff
@@ -161,6 +167,11 @@ export class ModalComponent implements OnInit, OnDestroy {
 	 */
 	ngOnInit() {
 		this.modalState = "in";
+		const primaryFocusElement  = this.modal.nativeElement.querySelector(this.selectorPrimaryFocus);
+		if (primaryFocusElement && primaryFocusElement.focus) {
+			primaryFocusElement.focus();
+			return;
+		}
```

Comment:
this should be run in an `ngAfterViewInit` since the view isn't guaranteed to exist until then.